test.py

```diff
@@ -363,6 +363,16 @@ def test_prefix_set(self):
         self.assertEqual([long_key], list(ps.iter(self._LONG_KEY)))
         self.assertEqual([other_key], list(ps.iter(self._OTHER_KEY)))
 
+    def testStringTrieSeparator(self):
+        """Separator tests."""
+        t = pygtrie.StringTrie(separator=",")
+        self.assertEqual(t._separator, ',')
+
+        t._separator = '.'
+        self.assertEqual(t._separator, '.')
+
+        self.assertRaises(Exception, lambda: pygtrie.StringTrie(separator=''))
```

Comment:
Or `self.assertRaises(ValueError, pygtrie.StringTrie, separator='')`. I personally prefer this but don’t mind either.